resources/public/api/conf/2.0/sandbox/income-sa-request-self-employments.md

```diff
@@ -11,45 +11,45 @@
     </thead>
     <tbody>
     <tr>
-        <td><p>Self-employments income data found</p></td>
-        <td><p>matchId=&lt;obtained from Individuals Matching API. example: 57072660-1df9-4aeb-b4ea-cd2d7f96e430&gt;</p><p>fromTaxYear=2018-18</p><p>toTaxYear=2019-20</p></td>
+        <td><p>A valid, successful request for self-employments income data</p></td>
+        <td><p>The matchID is obtained from the Individuals Matching API. For example: 57072660-1df9-4aeb-b4ea-cd2d7f96e430</p><p>fromTaxYear=2018-19</p><p>toTaxYear=2019-20</p></td>
```

Comment:
matchId should have lower case D - looks like this is incorrect on Lisa's doc, so will need updating there too.